### PR TITLE
Eliminate local CTCAE code system

### PR DESCRIPTION
**Pull request by saulakravitz**
1) Eliminate local CTCAE code system -- the publisher is currently not working well with NCIT terms.  This causes the expansion of the value set to be empty, and the example of an AE to induce an error.
2) Cleanups to artifacts list

### CODE REVIEW

File: input/pagecontent/StructureDefinition-codexrt-intrafraction-verification-intro.md
Comment (mvonsieb) [inline input/pagecontent/StructureDefinition-codexrt-intrafraction-verification-intro.md:1]: Why are all the intros deleted? Some where autogenerated for the examples, but at least this here and the ones on the main profiles had real information.

Comment (saulakravitz) [reply input/pagecontent/StructureDefinition-codexrt-intrafraction-verification-intro.md:1]: Obviously, I was a bit over-aggressive in my attempts to declutter the (now out of date) auto-generated stuff....restoring now....sorry for the sloppiness.

Comment (saulakravitz) [reply input/pagecontent/StructureDefinition-codexrt-intrafraction-verification-intro.md:1]: SHould now be restsored.

**Review by mvonsieb — APPROVED** [review]
Looks correct to me, but there are two new errors that we have to address or ignore: one link in the change log is broken because the page does not exist anymore: CodeSystem-codexrt-radiotherapy-adverse-event-severity-or-gradeCS.html, and looks like validation cannot find the NCIT code and therefore raises warnings and an error. https://build.fhir.org/ig/HL7/codex-radiation-therapy/branches/CTCAE/qa.html

**Comment by saulakravitz** [comment]
I fixed the broken link to the deleted codesystem (pointed to the STU1 version).
The NCI Thesaurus issues should go away.  I did a PR that includes all of the mCODE and CodexRT terms from NCIT that we use 
https://github.com/FHIR/packages/pull/29
https://github.com/FHIR/packages/pull/30
https://github.com/FHIR/packages/pull/31

**Pull request merged**